tests/lib/systemd.sh

```diff
@@ -10,6 +10,31 @@ systemd_create_and_start_unit() {
     systemctl start "$1"
 }
 
+# Create and start a persistent systemd unit that survives reboots. Use as:
+#   systemd_create_and_start_persistent_unit "name" "my-service --args"
+# The third arg supports "overrides" which allow to customize the service
+# as needed, e.g.:
+#   systemd_create_and_start_persistent_unit "name" "start" "[Unit]\nAfter=foo"
+systemd_create_and_start_persistent_unit() {
+    printf '[Unit]\nDescription=For testing purposes\n[Service]\nType=simple\nExecStart=%s\n[Install]\nWantedBy=multi-user.target\n' "$2" > "/etc/systemd/system/$1.service"
```

Comment:
If we could automatically insert the name of current test into the description, that would be a nice touch, perahps spread exposes it somehow?

Reply:
I like this a lot, will do a followup with this idea.